massa-consensus-worker/src/tests/scenarios_wishlist.rs

```diff
@@ -53,7 +53,7 @@ async fn test_wishlist_delta_with_empty_remove() {
                 &mut protocol_controller,
                 expected_new,
                 expected_remove,
-                1000,
+                cfg.t0.saturating_add(1000.into()).to_millis(), // leave 1sec extra for init and margin
```

Comment:
I'm not sure to understand how this line works, it's like 1s + 1s extra?

Reply:
the wishlist was supposed to be sent at slot (1,0), which happens t0 after genesis. But we waited exactly t0, so it would fail if program intialization or block emission took more than a couple ms. So here we leave t0 for the expected genesis period + 1 sec to account for boot time and message jitter